src/AvailabilityCheckerInterface.php

```diff
@@ -20,7 +20,7 @@
   public function applies(PurchasableEntityInterface $entity);
 
   /**
-   * Checks the availability of the given purchasable entity.
+   * Gets the availability of a given entity with the given context.
```

Comment:
We usually have the document block reflect the method name (this doesn't start with `getXXX`).

Maybe:

> Checks the availability of a given purchasable entity with given context.